src/components/styles/Date.styles.ts

```diff
@@ -1,10 +1,11 @@
 import { css } from "emotion";
+import { inputMarginBottom } from "./Input.styles";
```

Comment:
shouldn't this be `inputDropdownMarginBottom` ?

Reply:
Here this margin-bottom is set by input irrespective of whether that input is used in Select or DateInput.
So input is exporting the value and Dropdown components are using it.